app/views/layouts/application.slim

```diff
@@ -33,3 +33,9 @@ html lang="en"
     = javascript_include_tag :application
     - if Rails.env.production?
       = javascript_include_tag "//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-55ad6e555f728b0f", async: true
+    - unless Rails.env.test?
```

Comment:
The reasoning was the exact inverse. I don't know if there is a "sandboxed" setup for Intercom or something similar that we could be using to experiment in development and staging. The only thing I know is that currently no test at all benefits from this and we can just skip it.

Let's continue this discussion in https://github.com/subvisual/subvisual.co/pull/117#discussion_r47278176 and I'll resonate the results here afterwards.